play.html

```diff
@@ -331,21 +324,20 @@
 } 
 
 const (
-        NUMBERS = [1, 2, 3]
-
-        RED  = Color{r: 255, g: 0, b: 0}
-        BLUE = rgb(0, 0, 255)
+	NUMBERS = [1, 2, 3]
+	RED  		= Color{r: 255, g: 0, b: 0}
+	BLUE 		= rgb(0, 0, 255)
 )
 
 fn main() {
-        println(NUMBERS)
-        println(RED)
-        println(BLUE)
+	println(NUMBERS)
+	println(RED)
+  println(BLUE)
```

Comment:
Indentation issue

Reply:
I think we can merge it now... because of its looks not like this on me...